dis-segmentation/README.md

```diff
@@ -0,0 +1,104 @@
+# DIS Segmentation Truss
+
+This is a [Truss](https://truss.baseten.co/welcome) for [DIS](https://github.com/xuebinqin/DIS/tree/main)
+
+This model can be used to remove backgrounds from an image or create a segmentation mask for an object.
+
+This model takes an input image and creates two output images:
+1. This first output image is the original image with the background removed
+2. The second output image contains the mask of the object in the foreground
+
+
+## Deployment
+
+First, clone this repository:
+
+```sh
+git clone https://github.com/basetenlabs/truss-examples/
+cd dis-segmentation
+```
+
+Before deployment:
+
+1. Make sure you have a [Baseten account](https://app.baseten.co/signup) and [API key](https://app.baseten.co/settings/account/api_keys).
+2. Install the latest version of Truss: `pip install --upgrade truss`
+
+With `dis-segmentation` as your working directory, you can deploy the model with:
+
+```sh
+truss push
+```
+
+Paste your Baseten API key if prompted.
+
+For more information, see [Truss documentation](https://truss.baseten.co).
+
+### API route: `predict`
+
+The model only has one input:
+
+- `input_image` (required): An image represented as a base74 string.
+
+The output of the model contains two images in the form of a base64 string.
+Example model output:
+```json
+{
+    "img_without_bg": "Base64 string image",
+    "image_mask": "Base64 string image"
```

Comment:
is the mask optional?

Reply:
yeah, the mask is optional. But with background removal tasks, the mask comes in handy